### PR TITLE
Correct palette indexes of remap colors in image importer

### PR DESCRIPTION
**Pull request by zrowny**
The code that finds the closest entry that isn't used for remapping has the indices of remappable colors slightly off, and would make some colors in the imported image remappable even though they shouldn't have been.

The header file also has a comment that had the wrong number for the remappable color indices.

### CODE REVIEW

File: src/openrct2/drawing/ImageImporter.h [hunk 1]
```diff
@@ -273,7 +273,7 @@ constexpr const GamePalette StandardPalette = { {
     { 231, 231, 171, 255 },
     { 255, 255, 207, 255 },
 
-    // 203 - 214 (Secondary remap)
+    // 202 - 214 (Secondary remap)
```
Comment (Gymnasiast) [inline src/openrct2/drawing/ImageImporter.h:276]:
```suggestion
    // 202 - 213 (Secondary remap)
```

**Review by tupaschoal — COMMENTED** [review]
Nice catch :O

**Comment by Gymnasiast** [comment]
I wonder if this would solve https://github.com/OpenRCT2/OpenRCT2/issues/4896 .

**Comment by zrowny** [comment]
> I wonder if this would solve #4896 .

This made me realize there are actually different problems with the way this works. See #14649

**Pull request merged**